components/scream/src/physics/cld_fraction/atmosphere_cldfraction.cpp

```diff
@@ -0,0 +1,156 @@
+#include "atmosphere_cldfraction.hpp"
+
+#include "ekat/ekat_assert.hpp"
+#include "ekat/util/ekat_units.hpp"
+
+#include <array>
+
+namespace scream
+{
+  using namespace cldfrac;
+// =========================================================================================
+CldFraction::CldFraction (const ekat::Comm& comm, const ekat::ParameterList& params)
+ : m_cldfraction_comm (comm)
```

Comment:
Doesn't look like you need the comm, so you might as well not store it.

Reply:
It does look like I still need a `get_comm` getter function.  Since it is just an integer might as well store it so that the function makes sense.  See compile time error:

```
/usr/tce/packages/gcc/gcc-7.3.1/rh/usr/include/c++/7/ext/new_allocator.h(136): error: object of abstract class type "scream::CldFraction" is not allowed:
            pure virtual function "scream::AtmosphereProcess::get_comm" has no overrider
          detected during:
```

Reply:
Ah, shoot. Nobody is actually using that method, so far. I'm pretty sure I put it there for parallel splitting, but I can't remember.